ee-9/galleon-content-microprofile/pom.xml

```diff
@@ -0,0 +1,124 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  ~ JBoss, Home of Professional Open Source.
+  ~ Copyright 2021, Red Hat, Inc., and individual contributors
+  ~ as indicated by the @author tags. See the copyright.txt file in the
+  ~ distribution for a full listing of individual contributors.
+  ~
+  ~ This is free software; you can redistribute it and/or modify it
+  ~ under the terms of the GNU Lesser General Public License as
+  ~ published by the Free Software Foundation; either version 2.1 of
+  ~ the License, or (at your option) any later version.
+  ~
+  ~ This software is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+  ~ Lesser General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU Lesser General Public
+  ~ License along with this software; if not, write to the Free
+  ~ Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+  ~ 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+  -->
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.wildfly</groupId>
+        <artifactId>wildfly-ee-9-parent</artifactId>
+        <!--
+        Maintain separation between the artifact id and the version to help prevent
+        merge conflicts between commits changing the GA and those changing the V.
+        -->
+        <version>26.0.0.Beta1-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>wildfly-preview-feature-pack-content-microprofile</artifactId>
```

Comment:
This probably would be better if we name it differently. Instead of `content-microprofile`, maybe `content-full`.

This module is here just to supply the Galleon Full content that overrides the Galleon EE content. We could have content that is not MicroProfile specific, for example, at the moment of writing, we have here OIDC stuff which is not supplied by default by the EE Galleon Pack and it is not MicroProfile Specific.

Reply:
Same as above; seems ok for now.

Note that IIRC the oidc stuff will end up in wildfly-ee at some point.

Reply:
That is my impression too, I'm fine if we leave it as it is. Indeed, I think I was getting the relationships incorrectly. After discussions on https://issues.redhat.com/browse/WFLY-15752 I understand the general intention is:

wildfly-ee galleon pack -> known as "full", from "WildFly Full"
wildfly galleon pack-> known as microprofile, from "WildFly Micropofile"

Reply:
Yes, but TBH the word 'full' for wildfly-ee doesn't really fit any more either, since wildfly-servlet isn't really relevant any more and wildfly-ee isn't the "full distribution" of WF any more either, it's just a subset.  That 'full' word in things like standalone-full.xml comes from distinguishing the full EE platform vs the web profile. (But wildfly-servlet isn't EE web profile compliant either!) 

Your content-full idea isn't a bad idea; only negative is that the word 'full' now has such a confused meaning. Otherwise that would be a great name.

Reply:
Ok, got it, I'll resolve the conflicts to get it prepared